lib/releases.sh

```diff
@@ -7,6 +7,9 @@ function get_latest_release() {
   else
     release="$(curl -H "Authorization: token ${GITHUB_TOKEN}" -sL "${1}")" || ( set -x && exit 1 )
   fi
+  if [ "${EPHEMERAL_CLUSTER}" == "minikube" ]; then
+    sudo yum install jq -y
+  fi
```

Comment:
why is this here ? This file is only fetching the releases. if jq is not here, it means that it is sourced at the wrong place in 01_ . In addition, minikube != centos, you can run on Ubuntu with minikube

Reply:
Fixed.